_pages/dvntylr.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: dvntylr
+pageUrl: dvntylr.github.io
+timestamp: 2017-04-09
+---
```

Comment:
I see a strange character at the end of this line. I believe that is causing the issue!